src/nativeComponents/TableComponent/TableInner.tsx

```diff
@@ -73,6 +81,7 @@ const styles = () => ( {
     },
     title: {
         height: 40,
+        alignItems: 'center'
```

Comment:
not sure what css class is causing the problems, but there are problems if the screen width or screen height is small.

Reply:
Believe this was fixed with a recent flex-shrink: 0 rule I just added (if you're referring to the issue I saw)